api.md

```diff
@@ -2148,6 +2148,13 @@ WHERE table_schema='public' AND table_name='metrics';
 (1 row)
 ```
 
+If you want to see the current interval length for your hypertables, you can check the `_timescaledb_catalog` as follows. Note that for time-based interval lenghts, these are reported in microseconds.
+
+```Sql
+SELECT h.table_name, c.interval_length   FROM _timescaledb_catalog.dimension c
```

Comment:
Same as comment on previous PR w.r.t. SQL query line format 😸

Reply:
I thought I put this in line with the example from lines 2142 and 2143 - but happy to reformat in a different way. Should the ON be on a new line?